### DISCUSSION TITLE
How do I regenerate grub.cfg？

### DISCUSSION THREAD

**Discussion by bitcoffeeiux** [cobbler/cobbler#2655] [Version specific questions] [answered]
It doesn't seem to regenerate grub.cfg after I import a new profile.
https://cobbler.github.io/loaders/README no longer used and I used my own grubaa64.efi
version 3.2.0

**Comment by SchoolGuy** [comment, marked as answer]
The way I recommend the users are going forward is using the loaders supplied from the distro. For example if you are hosting libvirt you need the loaders anyway. For example in SUSE you can just install all loaders from all arches since they are noarch packages. I don't know how other distros handle it but I think that in any case Cobbler as a project can't help in building and supplying the bootloaders for the users due to things like security problems and other reasons...

**Reply by bitcoffeeiux** [reply]
> 我推荐用户前进的方式是使用发行版提供的加载程序。例如，如果您托管 libvirt，则无论如何都需要加载程序。例如，在 SUSE 中，您可以安装所有 arches 中的所有加载程序，因为它们是 noarch 包。我不知道其他发行版如何处理它，但我认为无论如何，由于安全问题和其他原因，作为一个项目的 Cobbler 无法帮助为用户构建和提供引导加载程序......

Thank you very much
